ui/client/containers/EspApp.js

```diff
@@ -95,15 +96,19 @@ class EspApp extends React.Component {
               <ul id="menu-items" className="nav navbar-nav navbar-right nav-pills nav-stacked">
                 <li><NavLink to={Processes.path}>{Processes.header}</NavLink></li>
                 <li><NavLink to={SubProcesses.path}>{SubProcesses.header}</NavLink></li>
-                {!_.isEmpty(this.props.featuresSettings.metrics) ?
-                  <li><NavLink to={Metrics.basePath}>{Metrics.header}</NavLink></li> : null}
-                {!_.isEmpty(this.props.featuresSettings.search) ?
-                  <li><NavLink to={Search.path}>{Search.header}</NavLink></li> : null }
-                {this.props.featuresSettings.signals ?
-                  <li><NavLink to={Signals.path}>{Signals.header}</NavLink></li> : null }
+                {
+                  !_.isEmpty(this.props.featuresSettings.metrics) ? <li><NavLink to={Metrics.basePath}>{Metrics.header}</NavLink></li> : null
```

Comment:
these lines are kinda long, maybe split them into three lines: condition, true case and false case?